lib/tasks/sandbox_data.rake

```diff
@@ -70,3 +70,14 @@ def create_school_and_associations(lead_provider, cohort, index)
 
   school
 end
+
+def random_trn
+  return if [true, false].sample
+
+  sprintf("%07i", Random.random_number(9_999_999))
+end
+
+def create_teacher
```

Comment:
Think it's about time we moved these into an included module. These are global scope for rake, so if any other rake task creates the same method it could cause spurious errors since they could overwrite each other.